### PR TITLE
school Library: Add classes

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Feature

- [ ] Create class Person with the following:
- [ ] Create class Student with the following:
- [ ] Create class Teacher with the following:

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require './person'
```
Comment (rloterh) [inline student.rb]: - [ ] [OPTIONAL]
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.

File: README.md [hunk 2]
```diff
@@ -1 +1,142 @@
...
+
+I would like to thank the Microverse community.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rloterh) [inline README.md:140]: - [ ] MIT License link is broken, kindly endeavor to fix it, to make ReadMe more professional!

**Broken ReadMe license link:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Manzi,  

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🌴  🏃 🌳 🏡 🏘️ 🌲 

## Highlights 👍  
🎲 Objects and inheritance principle properly constructed!
🎲 Properly encapsulated object properties!
🎲 ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @manzitresor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**